segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -489,4 +490,42 @@ public static void recoveryCompleted(long duration, int containerId) {
     }
 
     //endregion
+
+    //region ContainerEventProcessor
+
+    /**
+     * EventProcessor metrics.
+     */
+    public final static class EventProcessor implements AutoCloseable {
+
+        private final OpStatsLogger processingIterationLatency;
+
+        public EventProcessor(String processorName, int containerId) {
+            String[] eventProcessorTags = eventProcessorTag(containerId, processorName);
+            this.processingIterationLatency = STATS_LOGGER.createStats(MetricsNames.CONTAINER_EVENT_PROCESSOR_BATCH_LATENCY, eventProcessorTags);
+        }
+
+        public void batchProcessingLatency(long latency) {
+            this.processingIterationLatency.reportSuccessValue(latency);
+        }
+
+        @Override
+        public void close() {
+            this.processingIterationLatency.close();
+        }
+    }
+
+    /**
+     * Reports the outstanding bytes for a given {@link EventProcessor}.
+     *
+     * @param processorName      Name for the {@link EventProcessor}.
+     * @param containerId        Container id where the {@link EventProcessor} is running.
+     * @param outstandingBytes   Number of outstanding bytes for the {@link EventProcessor}.
+     */
+    public static void outstandingEventProcessorBytes(String processorName, int containerId, long outstandingBytes) {
+        DYNAMIC_LOGGER.reportGaugeValue(MetricsNames.CONTAINER_EVENT_PROCESSOR_OUTSTANDING_BYTES, outstandingBytes,
```

Comment:
Because we are showing here the current amount of outstanding bytes for this `EventProcessor` (a gauge). I think that this is more informative that using `OpStatsLogger`, which is basically a histogram/distribution of values.